projects/vdk-plugins/vdk-greenplum/.plugin-ci.yml

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-image: "python:3.7"
+
```

Comment:
nit: Is the extra line necessary (same comment for all the places with extra added lines)?